jenkins/run-server-func-tests

```diff
@@ -73,30 +88,25 @@ if [[ ${exit_status} -ne 0 ]]; then
     podman exec ${PB_POD_NAME}-pbenchserver journalctl
     printf -- "\n--- journalctl dump ---\n\n"
 fi
-if [[ -z ${1} ]]; then
-    echo "No clean up requested ... Pbench Server container and support services pod likely still running!"
+
+if [[ -z "${cleanup_flag}" ]]; then
+    echo "No clean up requested -- the Pbench Server container and support services pod are running!"
+    trap EXIT
```

Comment:
OK, granted that your explanation of intent makes sense: if we're only canceling the `EXIT` trap here, and we take a `SIGINT` or `SIGQUIT` here, as unlikely as that is, we'll duplicate your "no cleanup" message. One option would be to cancel *all* traps here.

Another is to simply call `exit` (without canceling any traps), which will skip the cleanup actions and generate the message. (Granted, it's a slightly different message since, without checking `${exit_status}`, here you're assuming everything *is* running instead of stating that it [is] "likely still running"...)

Reply:
> Another is to simply call `exit`

Yeah, I considered that, and I decided that I didn't like the message (and that I liked having two different messages for the two different cases, and that I didn't want to consult `$exit_status` in the cleanup handler to decide which to emit).

> One option would be to cancel all traps here.

Yes...and I briefly considered doing that when I was removing the `""`...I guess I should have acted on that.  I shall do so now.